Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextCharTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using Moq;
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextCharTests
+    {
+        [Fact]
+        public void NextChar_ShouldReturnChar()
+        {
+            var random = new Random();
+            var result = random.NextChar();
+
+            Assert.IsType<char>(result);
+        }
+
+        [Fact]
+        public void NextChar_ShoudReturnChar_CapitalLetter()
```

Comment:
1. Please use the following naming convention -> [MethodName]_[TestSetup]_Should[ExpectedResult]

2. Please merge all the NextChar tests into one, using XUnit Data Driven Tests using the [Theroy] and [MemberData] attributes

See: http://stackoverflow.com/questions/22093843/pass-complex-parameters-to-theory 
Note: PropertyData was replaced with MemberData in the newer XUnit versions, but the example is still applicable

For more detailed info, see: http://ikeptwalking.com/writing-data-driven-tests-using-xunit/